### DISCUSSION TITLE
How to fix when translate got problem "warning not found image"

### DISCUSSION THREAD

**Discussion by SavantEVO** [squidfunk/mkdocs-material#6346] [Q&A] [answered]
I'm experiencing an issue with my mkdocs project where images are not displaying correctly when I switch the language from the default (English) to Vietnamese. The problem arises because the URL adds a "vi" prefix, causing the image paths to break. Here are the details of the issue:

Steps to Reproduce:
Start the mkdocs serve command.
Access the default English version at: http://127.0.0.1:8000/ (images display correctly).
Switch the language to Vietnamese.
Access the Vietnamese version at: http://127.0.0.1:8000/vi (images are broken).

Expected Behavior:
Images should display correctly in both the default (English) and Vietnamese versions of the site.

Actual Behavior:
When switching to Vietnamese, the URL prefix "vi" is added, and images do not display because the path does not contain "vi/assets/images."

**Comment by alexvoss** [comment]
Hi, 

are you talking about your own images or images belonging to Material?

I assume you are using the static i18n plugin? There are [context variables](https://ultrabug.github.io/mkdocs-static-i18n/setup/using-i18n-context/) that would allow you to adapt any URLs in templates.  In the pages I guess you should adapt the URLs manually? That seems the straightforward solution to me.

If your question is about images produced by Material then that might be something for us to look at but we would need a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) to we have something to go be.

**Reply by SavantEVO** [reply]
I'm taking aboout my images. That meaning if english.md that is path image with prefix assets/images/pic.png and when I switch translate it process english.vi.md the path will be vi/assets/images/pic.png
So that should be

In my English Markdown file (`english.md`), the image path with the prefix `assets/images/`, like this:

```markdown
![Alt text](assets/images/pic.png)
```

In my Vietnamese Markdown file (`english.vi.md`), the image path with the prefix `vi/assets/images/`, like this:

```markdown
![Alt text](vi/assets/images/pic.png)
```
[Screencast from 16-11-2023 15:54:17.webm](https://github.com/squidfunk/mkdocs-material/assets/109493900/0bba7bb7-490f-48ab-b841-db18d7a02a7b)

![image](https://github.com/squidfunk/mkdocs-material/assets/109493900/d84d487d-ec74-4bc4-b493-9d5723cbe054)

So I have to create 1 folder with before prefix vi/... for process translate

**Reply by alexvoss** [reply, marked as answer]
OK, I can see that you are using the static i18n plugin. I assume you want the same images for both languages. The solution would be to use a relative path in the Vietnamese version that gets you out of the `vi` path and back to a path that does not include a language specifier. It might look something like this: `![Alt text](../assets/images/pic.png)` in a `.vi.md` file in `docs/`.

**Reply by SavantEVO** [reply]
**Oh I did it. Thank you so much !!!!**
